modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -2749,29 +2855,6 @@ private List<Object[]> _list(
 		return results;
 	}
 
-	private Map<String, Serializable> _putFormulaObjectFieldValues(
-			long objectDefinitionId, Map<String, Serializable> values)
-		throws PortalException {
-
-		List<ObjectField> objectFields =
-			_objectFieldLocalService.getObjectFields(objectDefinitionId);
-
-		for (ObjectField objectField : objectFields) {
-			if (!objectField.compareBusinessType(
-					ObjectFieldConstants.BUSINESS_TYPE_FORMULA)) {
-
-				continue;
-			}
-
-			ObjectFieldFormulaEvaluatorUtil.evaluate(
```

Comment:
@guilhermedcamacho do we still need this class `ObjectFieldFormulaEvaluatorUtil`?

Reply:
Yes, I think @seltong created it at the same time that I was refactoring this feature. He did it for this task "[LPS-168366 -
Formula fields don't work for custom fields or system fields in the system objects](https://issues.liferay.com/browse/LPS-168366)".
I will need to adjust my code to make things work also for system objects. Thanks.